src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlElementAttributes.cs

```diff
@@ -116,17 +117,17 @@ public void RemoveAt(int index)
 
         bool IList.IsReadOnly
         {
-            get { return InnerList.IsReadOnly; }
+            get { return false; }
```

Comment:
```InnerList``` is updated to ```List<>``` which didn't expose ```IsReadOnly``` property. Internally in ```List<>``` this property returns ```false```.